File: python/PRAK305-2210817310005-IkhsanGymnastiar.py
import math
bil = float(input("Masukan detik :"))
hari = bil / 86400
hasilsisa1  = bil % 86400
jam =  hasilsisa1 / 3600
hasilsisa2  = bil % 3600
menit =  hasilsisa2 / 60
detik = bil % 60
if bil >=0 and bil <= 59:
    print("00:00:", math.floor(bil))
elif bil >= 60 and bil <= 3599:
    print("00:{:.0f}:{:.0f}".format(math.floor(menit), math.floor(detik)))
elif bil >= 3600 and bil <= 86399:
    print("{:.0f}:{:.0f}:{:.0f}".format(math.floor(jam), math.floor(menit), math.floor(detik)))
else:
    print("{:.0f} hari {:.0f}:{:.0f}:{:.0f}".format(math.floor(hari), math.floor(jam), math.floor(menit), math.floor(detik)))

File: c/PRAK305-2210817310005-IkhsanGymnastiar.c
#include <stdio.h>
int main(){
    int bil, hari, jam, menit, detik, hasilsisa1, hasilsisa2;
    printf("Masukan detik :");scanf("%d", &bil);
    hari = bil / 86400;
    hasilsisa1 = bil % 86400;
    jam =  hasilsisa1 / 3600;
    hasilsisa2 = bil % 3600;
    menit =  hasilsisa2 / 60;
    detik = bil % 60;
    if(bil >=0 && bil < 60 ){
            printf("00:00:%d", detik);
    } else if(bil >= 60 && bil <= 3599){
            printf("00:%d:%d", menit, detik);
    } else if(bil >= 3600 && bil <= 86399){
          printf("%d:%d:%d", jam, menit, detik);
    } else{
        printf("%d hari %d:%d:%d", hari, jam, menit, detik);
    }
    return 0;
}
